crates/forge/bin/cmd/doc/server.rs

```diff
@@ -0,0 +1,104 @@
+use axum::{routing::get_service, Router};
+use forge_doc::mdbook::{utils::fs::get_404_output_file, MDBook};
+use std::{
+    net::{SocketAddr, ToSocketAddrs},
+    path::PathBuf,
+};
+use tower_http::services::{ServeDir, ServeFile};
+
+/// The HTTP endpoint for the websocket used to trigger reloads when a file changes.
+const LIVE_RELOAD_ENDPOINT: &str = "/__livereload";
```

Comment:
oh hell yeah,
I think for reloading we'd need a watch service that runs forge doc?

Reply:
This was half-implemented. I can open a follow up implementing this

Reply:
sweet,

I can't really remember how the watchexec API works, but I believe it was registering an action via a closure, maybe the forge watch command is helpful, perhaps not